python/tvm/auto_scheduler/utils.py

```diff
@@ -157,6 +157,9 @@ def __reduce__(self):
 
 def kill_child_processes(parent_pid, sig=signal.SIGTERM):
     """kill all child processes recursively"""
+    if not psutil:
+        raise ImportError("psutil not found, try `pip install psutil` to fix this")
```

Comment:
Ah...you move it here. But i am curious why we should need it here. For testing?

Reply:
Yes, it is for testing. The goal is to minimize the dependency to run tvm.